omod/src/main/webapp/pages/login.gsp

```diff
@@ -9,6 +9,8 @@
     <title>${ ui.message("referenceapplication.login.title") }</title>
     <link rel="shortcut icon" type="image/ico" href="/${ ui.contextPath() }/images/openmrs-favicon.ico"/>
     <link rel="icon" type="image/png\" href="/${ ui.contextPath() }/images/openmrs-favicon.png"/>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
```

Comment:
@Akayeshmantha Does this make boot strap available even without internet?  cc @dkayiwa